src/main/webapp/community.html

```diff
@@ -16,8 +16,8 @@
           userlist.innerHTML = '';
 
           users.forEach((user) => {
-          const userListItem = buildUserListItem(user);
```

Comment:
Hmmmmm, this seems highly suspect to me. Do you have any more details on what the error is without these white spaces?